### ISSUE TITLE
Reproducibility on Resnet-110 on cifar-100

### ISSUE THREAD

**Issue by GuoleiSun** [clovaai/CutMix-PyTorch#10]
Hi, I run your code on cifar-100 using Resnet-110 because PyramidNet-200 takes long time to finish. I used the default parameters as PyramidNet-200. Here is what I got:
try1: 20.23 4.65
try2: 21.3 4.93

However, 20.11, 4.43 are reported in the paper (in Table 6). May I know what settings did you use for Resnet-110?

**Comment by hellbell** [comment]
@GuoleiSun 
Thank you for having interests in our work!

We used 2 GPUs with a training setting for `ResNet-110 + CutMix` as:
```
--depth 110 
--batch_size 64 
--lr 0.1
--cutmix_prob 1.0
--beta  1.0
```

Our ResNet-110 experiment logs:

 \ | top-1 | top-5
-- | -- | --
try1 | 20.52 | 4.40
try2 | 20.10 | 4.43
try3 | 19.72 | 4.46
average | 20.11 | 4.43

Hope this works!

**Comment by GuoleiSun** [comment]
Oh, you used learning rate of 0.1, instead of 0.25, for Resnet-110+cutmix. I will try 0.1 and let you know the results soon.
